copy/copy.go

```diff
@@ -250,13 +262,19 @@ func Image(ctx context.Context, policyContext *signature.PolicyContext, destRef,
 	}
 
 	if !multiImage {
+		if len(options.EnsureCompressionVariantsExist) > 0 {
+			return nil, fmt.Errorf("EnsureCompressionVariantsExist is not implemented for copySingleImage")
```

Comment:
Something like
```suggestion
			return nil, fmt.Errorf("EnsureCompressionVariantsExist is not implemented when not creating a multi-architecture image")
```
please, users don’t see the private `copySingleImage` name.

In both places.

Reply:
done :)